### DISCUSSION TITLE
Solid HT3D cells after complete decomposition

### DISCUSSION THREAD

**Discussion by Vojta-Salek** [firemodels/fds#13587] [Q&A]
Dear FDS community,

Do you have any insights into what happens to a solid non-charring pyrolyzing cell in `HT3D='T'` mode after all the material has decomposed? Assume `BURN_AWAY` is not specified, as it is currently unavailable for `HT3D`.

For the default 1D heat conduction: "_If `BURN_AWAY` is not prescribed, the surface temperature is set to `TMP_GAS_BACK`, convective heat flux to zero and burning rate to zero_" (User’s Guide Sec. 8.3.8).  Does the same happen in `HT3D` that `TMP_GAS_BACK` is assigned to all surfaces? If so, I assume the correct `BACKING` is applied for all surfaces (each surface being either exposed to gas phase or to a neighboring `HT3D` solid cell according to the limitations in Sec. 8.4.2).

In the very simple FDS case provided below, we can observe, for instance, negative surface temperatures (how?) or extremely high surface temperatures (which seems expected for an object of nearly zero density). However, these surface temperatures in technically "dead" cells might significantly influence other parts of the computational domain (due to radiation), even if convective heat flux is artificially set to zero. It has also caught my attention, that although the proposed model is symmetrical, surface temperatures of “dead” cells seem random (see [Smokeview screen](https://github.com/Vojta-Salek/Pictures/blob/main/1_Symmetrical_surface.PNG)).

[Negative_temperature.txt](https://github.com/user-attachments/files/17405429/Negative_temperature.txt)
[Negative_temperature_devc.csv](https://github.com/user-attachments/files/17405330/Negative_temperature_devc.csv)
[Display_plot.py.txt](https://github.com/user-attachments/files/17405915/Display_plot.py.txt) (Python for quick data plotting)

![1_Negative_tempearature](https://github.com/user-attachments/assets/b0de4d3b-e60b-4b60-87b3-fece54fc9e8a)

Could someone please explain a bit more about what is actually happening? What effect does this have on neighboring solid cells? Does this behavior compromise calculations once “dead” cells occur in domain, or is it more of an issue with the visualization of surface temperatures? I am aware that `HT3D` is still in development, therefore I'm not submitting an issue, but rather seeking clarification on the current behavior of "dead" material.

Thank you in advance for any insights into this topic!

Sincerely,

Vojtěch

**Comment by mcgratta** [comment]
I'll take a look at your case.

**Comment by mcgratta** [comment]
This is a good example of the limitations of the 3D solver. The 3D algorithm in FDS is an extension of the 1D, where each surface cell performs a 1D heat conduction calculation from front to back. The 3D feature allows some of the heat to be conducted in the two lateral directions. Even though the 1D conduction paths shrink as the fuel is consumed, the 3D lateral heat transfer continues under the original pattern. This means that is this thing shrinks, the 3D aspect becomes less accurate. 

One thing I will check in this case is conservation of mass and energy. If these are not satisfied, I'll focus attention on that.

**Comment by mcgratta** [comment]
I'm going to make this an issue so that I do not lose track of it. The large drop in temperature is a bug. I suspect it is related to having internal nodes disappear, which is the normal way of things for the 1-D heat conduction algorithm. When the 1-D nodal structure changes, I do not believe that the 3-D algorithm is working.

**Comment by mcgratta** [comment]
Here is the [issue](https://github.com/firemodels/fds/issues/13597)
